source/account_structure/index.html.md.erb

```diff
@@ -1,92 +1,68 @@
 ---
-title: How accounts work
-last_reviewed_on: 2021-01-08
+title: Setting up multiple services
+last_reviewed_on: 2021-02-03
 review_in: 6 months
 weight: 40
 ---
 
-# How accounts work
+# Setting up multiple services
+  
+If you have more than one online service, you can set up your GOV.UK Pay account and payment service provider (PSP) accounts so it's easier to manage your payments and services.
 
-GOV.UK Pay does not limit how many services you can integrate with it and how many accounts you can have with us.
-GOV.UK Pay test and live accounts are free to create and maintain but your payment service provider may charge you for the underlying accounts.
+## Decide where to send users
 
-## Account structure examples
+You can send your users to make payments with either:
 
-If you do integrate multiple services, you can treat them separately or as a
-combined single service within GOV.UK Pay and your payment service provider (PSP).
-This gives 3 possible account configurations.
+- a different GOV.UK Pay service for each of your services
+- a combined GOV.UK Pay service
 
-As an example, take 2 services called ‘parking permits’ and ‘parking fines’:
+### Send your users to different GOV.UK Pay services
 
-### Type 1 (GOV.UK Pay separate, PSP separate)
+Send your users to a different GOV.UK Pay service for each of your services if you need to do any of the following:
 
-<%= image_tag "/images/accountstructure_1.svg", { :alt => '' } %>
+- give each service its own name on the GOV.UK Pay payment pages
+- report on each service's payments separately
+- send each service's payments to different bank accounts
+- give your staff members access to each service separately in the GOV.UK Pay admin tool
 
-The diagram show two services, ‘Parking permits’ and ‘Parking fines’, that are considered separate services by
-both GOV.UK Pay and the PSP.
+To set up your services like this, [create a new GOV.UK Pay service](https://selfservice.payments.service.gov.uk/my-services/create) for each of your online services.
 
-If you use this account structure, you can:
+GOV.UK Pay services are free, and there's no limit to how many you can create.
```

Comment:
Maybe we need to say, if you have lots of services set up in Pay, you'll need to manage settings on each one. Basically want to remind them that there is an overhead to managing lots of services individually - needing to add/remove users when they leave, needing to change settings on each one if they change what card types they accept. sort of referenced in the section below but I have a feeling it's slightly overlooked as a long term consideration

Reply:
There's also something we could say about the granularity of services... (thugh maybe this gets too complicate for round 1 of improvements, or is best shown in an example...)

An LA may want a 'registrar office' service, and within that they can have API and/or payment links for each type of certificate - marriage, birth etc. The can use our reporting features to differentiate between payments, and everyone in the registrar office can see all the transactions.  They then set up a 'Parking' service, and a 'Garden Waste' service etc, reflecting different team areas.

But some LAs are setting up a 'birth certificate' service, and a 'marriage certificate' service. My guess is they all have the same users, the same settings (though actually, that's something to check!) In which case I think they are doing this for reporting reasons...? but there's an overhead - everytime someone leaves the team they need to remove them for 10 services instead of 1. etc. (maybe that's minimal, but feels inefficient). \

Anyways that is maybe too much info, but I do think knowing where to draw the line in granularity is an issue for services...

Reply:
On the first point: see if this looks better in 81e419d. Now it's broken down a bit more, it should hopefully be clearer to users what the advantages/disadvantages of each approach are.

On the second point: I agree that granularity of services may over-complicate things at this stage, so fully going into this is probably out of scope. However I've gone partway there by attempting to add a bit about using different API keys/payment links but I'm not sure I have the details right, so please do suggest improvements.